src/NuGet.Clients/VisualStudioAPI/IPackageManagerProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Interface for integrating other package manager.eg:bower,npm
+    /// Contains methods to check packages in intergated package manager and direct to intergated package manager. 
+    /// </summary>
+    [ComImport]
+    [Guid("BCED5BF2-40FC-4D9F-BF0A-43CD4E9FF65F")]
+    public interface IPackageManagerProvider
+    {
+        /// <summary>
+        /// Package manager name.
```

Comment:
Yes, it's localized. it will return something like Resources.Strings.BowerPackageProviderName from integrated Manager